tooling/cli/templates/mobile/ios/project.yml

```diff
@@ -97,19 +97,16 @@ targets:
         embed: false
       {{~#each ios-libraries}}
       - framework: {{this}}
-        embed: false{{/each}}
-      {{~#each ios-vendor-frameworks}}
-      - framework: {{prefix-path this}}{{/each}}
-      {{~#each ios-vendor-sdks}}
-      - sdk: {{prefix-path this}}{{/each}}
+        embed: false{{/each}}{{#if ios-vendor-frameworks}}{{~#each ios-vendor-frameworks}}
+      - framework: {{this}}{{/each}}{{/if}}{{#if ios-vendor-sdks}}{{~#each ios-vendor-sdks}}
+      - sdk: {{prefix-path this}}{{/each}}{{/if}}
       - sdk: CoreGraphics.framework
-      - sdk: Metal.framework
+      - sdk: Metal.frameworks
```

Comment:
Is it framework or frameworks 👀

Reply:
I have no idea how i added this 😂 thanks